automated-testing/container-testing/azdo-container-dev-test-release.md

```diff
@@ -0,0 +1,286 @@
+# Building Containers with Azure DevOps using DevTest Pattern
```

Comment:
Thanks for sharing your learnings. I like this, I think we should add a new section for case studies/examples and place this there.